model/github_runner.rb

```diff
@@ -46,7 +46,7 @@ def runner_url
   def log_duration(message, duration)
     values = {ubid:, label:, repository_name:, duration: duration.round(3), conclusion: workflow_job&.dig("conclusion")}
     if vm
-      values.merge!(vm_ubid: vm.ubid, arch: vm.arch, cores: vm.cores, vcpus: vm.vcpus)
+      values.merge!(vm_ubid: vm.ubid, arch: vm.arch, cores: vm.cores, vcpus: vm.vcpus, boot_image: vm.vm_storage_volumes.first&.boot_image&.version || vm.boot_image)
```

Comment:
Why the `&.version` for the `vm.vm_storage_volumes.first&.boot_image` case but not the fallback case? Seems like you would want either:

```suggestion
      values.merge!(vm_ubid: vm.ubid, arch: vm.arch, cores: vm.cores, vcpus: vm.vcpus, boot_image: (vm.vm_storage_volumes.first&.boot_image || vm_boot_image)&.version)
```

or:

```suggestion
      values.merge!(vm_ubid: vm.ubid, arch: vm.arch, cores: vm.cores, vcpus: vm.vcpus, boot_image: vm.vm_storage_volumes.first&.boot_image || vm_boot_image)
```